ovs-service/src/main/java/org/dcsa/ovs/mapping/VesselScheduleMapper.java

```diff
@@ -0,0 +1,14 @@
+package org.dcsa.ovs.mapping;
+
+import org.dcsa.ovs.persistence.entity.Vessel;
+import org.dcsa.ovs.transferobjects.VesselScheduleTO;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {TransportCallMapper.class})
+public interface VesselScheduleMapper {
```

Comment:
Since mapstruct only maps fields that have exactly the same name, maybe put in a FieldValidator test on this